portfolio/src/main/webapp/style.css

```diff
@@ -9,17 +10,19 @@
 }
 
 body {
-    background-color: palegoldenrod;
+  background-color: purple;
```

Comment:
lol NBD, but purple is WAY more bold than light yellow 🤣

Reply:
DEFINITELY! LOL. Lakers are my favorite team, and when you highlight a section on my navbar it turns yellow!

Reply:
oooh cool!